charts/fluid/fluid/values.yaml

```diff
@@ -33,8 +33,8 @@ runtime:
     portRange: 18000-19999
     enabled: false
     smartdata:
-      image: registry.cn-shanghai.aliyuncs.com/jindofs/smartdata:3.5.2
+      image: registry.cn-shanghai.aliyuncs.com/jindofs/smartdata:3.5.0
```

Comment:
3.5.0 is older than 3.5.2? Is it your expectation?

Reply:
thx，The version of the mirror will be same as  JindoFS version in the future，so 3.5.0 is expected